api/go.sum

```diff
@@ -1,30 +1,86 @@
+cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Need to confirm the license of these new dependencies

Reply:
License of all packages are checked, everything is OK.

- github.com/cameront/go-jsonpatch MIT
- github.com/coreos/bbolt MIT
- github.com/dustin/go-humanize MIT
- github.com/golang/groupcache Apache 2.0
- github.com/grpc-ecosystem/go-grpc-middleware Apache 2.0
- github.com/grpc-ecosystem/grpc-gateway BSD 3.0
- github.com/jonboulle/clockwork Apache 2.0
- github.com/prometheus/client_golang Apache 2.0
- github.com/spf13/cobra Apache 2.0
- github.com/tmc/grpc-websocket-proxy MIT
- golang.org/x/time BSD 3.0
- sigs.k8s.io/yaml MIT

Reply:
nice